lighthouse-core/test/gather/gatherers/source-maps-test.js

```diff
@@ -48,10 +48,8 @@ describe('SourceMaps gatherer', () => {
     const sendCommandMock = createMockSendCommandFn()
       .mockResponse('Debugger.enable', {})
       .mockResponse('Debugger.disable', {})
-      .mockResponse('Network.enable', {})
-      .mockResponse('Fetch.enable', {})
-      .mockResponse('Fetch.disable', {});
-    const fetchMock = fnAny();
+      .mockResponse('Network.enable', {});
+    const fetchMock = jest.fn();
```

Comment:
Does it matter if this is `jest.fn()` or `fnAny()`?

Reply:
it's fine here because the file is not type checked.